### PR TITLE
[hi] Create vertical-scaling.md

### LINKED ISSUE #2624: [hi] Localize Vertical Scaling
localize the https://glossary.cncf.io/vertical-scaling/ for Hindi localization folks

### PR DESCRIPTION
**Pull request by Aryan-Singla**
Signed off: Aryan-Singla [devendergupta4573@gmail.com]
### Describe your changes
 Localize Vertical Scaling in Hindi

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2624

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
@ARYANis0001 I have a common comment on your PRs.
PTAL: https://github.com/cncf/glossary/pull/2605#issuecomment-1815774999

**Comment by kumarankit999** [comment]
+1 @seokho-son

**Comment by ARYANis0001** [comment]
> +1 @seokho-son 


Hey @seokho-son  
Hey @kumarankit999 
I want to express my sincere apologies for any inconvenience caused to the team members of CNCF/Glossary due to my raised PRs. I fully understand my mistake and acknowledge that I need to be more careful in the future. Going forward, I will ensure to thoroughly read the contributing guidelines and submit PRs in my native language, which I am fluent in and proficient in.

**Review by ARYANis0001 — COMMENTED** [review]
Update (add some words)

**Comment by ARYANis0001** [comment]
Hey @aj11anuj @kumarankit999  I have added some words to simplify it can you please review it

File: content/hi/vertical-scaling.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+जैसे-जैसे किसी एप्लिकेशन की मांग उस एप्लिकेशन इंस्टेंस की वर्तमान क्षमता से अधिक बढ़ती है, 
+तो हमें सिस्टम को स्केल करने (क्षमता जोड़ने) का एक तरीका खोजने की जरूरत है।
+हम या तो मौजूदा नोड्स में अधिक गणना संसाधन जोड़ सकते हैं (ऊर्ध्वाधर स्केलिंग)
+या सिस्टम में अधिक नोड्स ([क्षैतिज स्केलिंग](/क्षैतिज-स्केलिंग/))।
+[स्केलेबिलिटी](/स्केलेबिलिटी/) प्रतिस्पर्धात्मकता, दक्षता, प्रतिष्ठा और गुणवत्ता में योगदान करती है।
+
+## यह कैसे मदद करता है
+
+वर्टिकल स्केलिंग आपको एप्लिकेशन कोड को बदले बिना अपने सर्वर का आकार बदलने की अनुमति देता है।
+यह क्षैतिज स्केलिंग के विपरीत है, जहां ऐप को स्केल करने योग्य होना चाहिए, जहां संभावित रूप से कोड अपडेट की आवश्यकता होती है।
```
Comment (ankur0904) [inline content/hi/vertical-scaling.md:27]:
```suggestion
यह क्षैतिज स्केलिंग के विपरीत है, जहां ऐप को स्केल करने के लिए प्रतिकृति चाहिए, जहां संभावित रूप से कोड अपडेट की आवश्यकता होती है।
```

**Review by ankur0904 — CHANGES_REQUESTED** [review]
Please have a look at this suggestion.

**Comment by Aryan-Singla** [comment]
> Please have a look at this suggestion.

Updated

**Review by ankur0904 — APPROVED** [review]
LGTM

**Comment by Aryan-Singla** [comment]
Hey @aj11anuj @kumarankit999  please review it

**Review by Garima-Negi — APPROVED** [review]
LGTM!

**Comment by Aryan-Singla** [comment]
@bishal7679 why this hold label is added

File: content/hi/vertical-scaling.md (hunk 1 above)
Comment (Aryan-Singla) [reply content/hi/vertical-scaling.md]: Added the suggestions

**Review by abhay-raj19 — APPROVED** [review]
LGTM ! 
Thanks for contributing .

**Pull request merged**